lib/graphql/transaction.ts

```diff
@@ -86,27 +86,27 @@ export class Transaction extends IterableModel<TransactionModel> {
 
     const pageInfo = result?.viewer?.mainAccount?.transactions?.pageInfo ?? {
       hasNextPage: false,
-      hasPreviousPage: false
-    };
+      hasPreviousPage: false,
+    }           ;
```

Comment:
This looks a bit weird 🤔 is it a typo or something?

Reply:
Good catch! Yes - initially I used an old version of tslint which did not knew about the used nullish coalescing operator above and it created that silly whitespace.
Found and fixed one more occurrence.